ltm/EntityMetadata meta

uint8 n_people    #  Size of the crowd
uint8 n_male      #  Number of male people
uint8 n_female    #  Number of female people
uint8 n_children  #  Number of children
uint8 n_adults    #  Number of adults
uint8 n_elders    #  Number of elders

string location       